homeassistant/components/system_bridge/binary_sensor.py

```diff
@@ -26,7 +26,26 @@ class SystemBridgeBinarySensorEntityDescription(BinarySensorEntityDescription):
     value: Callable = round
```

Comment:
Please rename this to value_fn since it's a function, that should be done in a separate PR though.
```suggestion
    value_fn: Callable = round
```